server/src/main/java/chat/database/UsersBase.java

```diff
@@ -0,0 +1,69 @@
+package chat.database;
+
+import chat.model.User;
+import org.apache.log4j.Logger;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UsersBase {
+
+    private static final Logger LOGGER = Logger.getLogger(UsersBase.class);
+    private static final String URL = "jdbc:mysql://localhost:3306/Users?serverTimezone=UTC";
+    private static final String NAME_BASE = "root";
+    private static final String PASS_BASE = "root123321";
```

Comment:
You should extract your properties to properties file

Reply:
++